data/sql_partition/partition_schedule_b.sql

```diff
@@ -0,0 +1,94 @@
+    cand_nm_last                  VARCHAR(38),
+    cand_m_nm                     VARCHAR(20),
+    cand_prefix                   VARCHAR(10),
+    cand_suffix                   VARCHAR(10),
+    cand_office                   VARCHAR(1),
+    cand_office_desc              VARCHAR(20),
+    cand_office_st                VARCHAR(2),
+    cand_office_st_desc           VARCHAR(20),
+    cand_office_district          VARCHAR(2),
+    disb_dt                       TIMESTAMP,
+    disb_amt                      NUMERIC(14,2),
+    memo_cd                       VARCHAR(1),
+    memo_cd_desc                  VARCHAR(50),
+    memo_text                     VARCHAR(100),
+    disb_tp                       VARCHAR(3),
+    disb_tp_desc                  VARCHAR(90),
+    conduit_cmte_nm               VARCHAR(200),
+    conduit_cmte_st1              VARCHAR(34),
+    conduit_cmte_st2              VARCHAR(34),
+    conduit_cmte_city             VARCHAR(30),
+    conduit_cmte_st               VARCHAR(2),
+    conduit_cmte_zip              VARCHAR(9),
+    national_cmte_nonfed_acct     VARCHAR(9),
+    ref_disp_excess_flg           VARCHAR(1),
+    comm_dt                       TIMESTAMP,
+    benef_cmte_nm                 VARCHAR(200),
+    semi_an_bundled_refund        NUMERIC(14,2),
+    action_cd                     VARCHAR(1),
+    action_cd_desc                VARCHAR(15),
+    tran_id                       TEXT,
+    back_ref_tran_id              TEXT,
+    back_ref_sched_id             TEXT,
+    schedule_type                 VARCHAR(2),
+    schedule_type_desc            VARCHAR(90),
+    line_num                      VARCHAR(12),
+    image_num                     VARCHAR(18),
+    file_num                      NUMERIC(7,0),
+    link_id                       NUMERIC(19,0),
+    orig_sub_id                   NUMERIC(19,0),
+    sub_id                        NUMERIC(19,0) NOT NULL,
+    filing_form                   VARCHAR(8) NOT NULL,
+    rpt_tp                        VARCHAR(3),
+    rpt_yr                        NUMERIC(4,0),
+    election_cycle                NUMERIC(4,0),
+    timestamp                     TIMESTAMP,
+    pg_date                       TIMESTAMP,
+    pdf_url                       TEXT,
+    recipient_name_text           TSVECTOR,
+    disbursement_description_text TSVECTOR,
+    disbursement_purpose_category TEXT,
+    clean_recipient_cmte_id       VARCHAR(9),
+    two_year_transaction_period   SMALLINT
+);
+
+-- Create the child tables.
+SELECT create_itemized_schedule_partition('b', 1978, 2018);
+SELECT finalize_itemized_schedule_b_tables(1978, 2018);
+
+-- Create the insert trigger so that records go into the proper child table.
+DROP TRIGGER IF EXISTS insert_sched_b_trigger_tmp ON ofec_sched_b_master_tmp;
+CREATE trigger insert_sched_b_trigger_tmp BEFORE INSERT ON ofec_sched_b_master_tmp FOR EACH ROW EXECUTE PROCEDURE insert_sched_master('ofec_sched_b_');
+
+SELECT rename_table_cascade('ofec_sched_b_master');
```

Comment:
Flagging this again for the same remarks as schedule A as I know this part is still being worked on.